app-containers/skopeo/files/makefile-1.13.3.patch

```diff
@@ -0,0 +1,31 @@
+--- a/Makefile
++++ b/Makefile
+@@ -159,25 +159,23 @@
+ 	rm -rf bin docs/*.1 completions/
+ 
+ install: install-binary install-docs install-completions
+ 	install -d -m 755 ${DESTDIR}${LOOKASIDEDIR}
+ 	install -d -m 755 ${DESTDIR}${CONTAINERSCONFDIR}
+-	install -m 644 default-policy.json ${DESTDIR}${CONTAINERSCONFDIR}/policy.json
```

Comment:
Why do we have to remove these configs from skopeo?

Reply:
because these files are installed by newly added dep, containiss-common. If we install by skopeo then it'll conflict.

Reply:
I think William at least wasn't clear on why we're doing containers-common. But it's because of https://bugs.gentoo.org/849863 right? I'm thinking if we put an explanation somewhere, it'll help people who are wondering.